### PR TITLE
Add on'event' handler tests for pointerevents

### PR DESCRIPTION
**Pull request by NavidZ**
Adds tests for w3c/pointerevents#157

### CODE REVIEW

File: pointerevents/pointerevent_on_event_handlers.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                for (const location of [window, HTMLElement.prototype, SVGElement.prototype, Document.prototype]) {
+                  assert_true(location.hasOwnProperty(on_event),
+                    `${location.constructor.name} has an own property named `+ on_event);
+                }
+            }, on_event + " is on the appropriate locations for GlobalEventHandlers");
+            test(function {
+              const htmlElement = document.createElement("span");
+              const svgElement = document.createElementNS("http://www.w3.org/2000/svg", "g");
+              for (const location of [window, htmlElement, svgElement, document]) {
+                assert_equals(location.onauxclick, null,
```
Comment (dtapuska) [inline pointerevents/pointerevent_on_event_handlers.html]: onauxclick?

File: pointerevents/pointerevent_on_event_handlers.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+              compiledHandler();
+              assert_equals(window.eventHappened, event, "Calling the handler must run the code");
+            }, "The " + on_event + " content attribute must be compiled into the " + on_event + " property");
+
+            test(function {
+              const element = document.createElement("meta");
+              element[on_event] = function(e) {
+                assert_equals(e.currentTarget, element, "The event must be fired at the <meta> element");
+              };
+              element.dispatchEvent(new Event(event));
```
Comment (dtapuska) [inline pointerevents/pointerevent_on_event_handlers.html]: How does this assert that the handler did actually fire?

File: pointerevents/pointerevent_on_event_handlers.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            }, "dispatching an " + on_event + " event must trigger element." + on_event);
```
Comment (dtapuska) [inline pointerevents/pointerevent_on_event_handlers.html]: shouldn't this be |event| and not |on_event| for the first param?

File: pointerevents/pointerevent_on_event_handlers.html (hunk 2 above)
Comment (NavidZ) [inline pointerevents/pointerevent_on_event_handlers.html]: Yeah. Makes sense. I changed it to an async test so if the handler doesn't fire that test times out.

**Review by dtapuska — APPROVED** [review]

File: pointerevents/pointerevent_on_event_handlers.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    ></div>
+    <script>
+        window.eventHappened = '';
+
+        All_Pointer_Events.forEach(function(event) {
+            var on_event = "on" + event;
+
+            test(function() {
+                for (const location of [window, HTMLElement.prototype, SVGElement.prototype, Document.prototype]) {
+                  assert_true(location.hasOwnProperty(on_event),
```
Comment (RByers) [inline pointerevents/pointerevent_on_event_handlers.html]: Nit: I think this sort of test is better done by an IDL test - proposed [here](https://github.com/w3c/web-platform-tests/pull/4358).  I'd suggest removing the existence checks, but keeping the functionality checks.

**Review by RByers — CHANGES_REQUESTED** [review]

**Review by RByers — APPROVED** [review]

**Pull request merged**